CanDIG/query
# DIG-1372: Query can assemble discovery queries

# Description
[DIG-1372](https://candig.atlassian.net/browse/DIG-1372)

This creates another new endpoint: `/discovery/query`, that takes all the same arguments and logic as `/query` but returns counts only, and does so with elevated priviledges.

# To Test:
1. Test the endpoint with no authorization: `curl candig.docker.internal:1236/discovery/query`
2. Add a primary site and ensure that it returns just the donor with that (1 result): `curl candig.docker.internal:1236/discovery/query?primary_site=Esophagus`
3. Add multiple primary sites and ensure it returns donors with either one (2 results): `curl "candig.docker.internal:1236/discovery/query?primary_site=Esophagus&primary_site=Gallbladder"`
4. Add multiple clinical parameters and ensure it correctly parses the donor that has both (1 result): `curl "candig.docker.internal:1236/discovery/query?primary_site=Gallbladder&treatment=Immunotherapy"`
5. Add a genomic parameter and ensure it correctly parses it: (1 result): `curl candig.docker.internal:1236/discovery/query?chrom=chr21:5030000-5030847`
6. Add both, make sure it parses that properly

[DIG-1372]: https://candig.atlassian.net/browse/DIG-1372?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
